mfusepy.py

```diff
@@ -1161,7 +1161,7 @@ class FUSE:
         ('nothreads', '-s'),
     )
 
-    def __init__(self, operations, mountpoint: str, raw_fi: bool = False, encoding: str = 'utf-8', **kwargs) -> None:
+    def __init__(self, operations, mountpoint: str, raw_fi: bool = False, encoding: str = 'utf-8', errors: str = 'strict', **kwargs) -> None:
```

Comment:
`errors` feels pretty generic. Maybe something like `encoding_errors`? Then again, it seems a bit verbose. I don't have a strong opinion on this and would be fine to merge as is.

Reply:
well, had the same feelings, but then kept errors because the argument to .decode/.encode is named like that.

also, when using something longer, some already long lines with encode/decode get even longer.

Reply:
hmm, how about:
```
self.coding_policy = ("utf-8", "surrogateescape")
...
foo.decode(*self.coding_policy)
```